perception/object_recognition/detection/tensorrt_yolo/README.md

```diff
@@ -1,4 +1,126 @@
-# Tensorrt YOLO
+# tensorrt_yolo
+
+## Purpose
+
+This package detects 2D bounding boxes for target objects e.g., cars, trucks, bicycles, and pedestrians on a image based on YOLO(You only look once) model.
+
+## Inner-workings / Algorithms
+
+### Cite
+
+yolov3
+
+```bibtex
```

Comment:
IMHO: bibtex format is useful for researchers who write papers using TeX. For other people, the usual reference style might be easier to read. For example:

- Redmon, J., & Farhadi, A. (2018). Yolov3: An incremental improvement. arXiv preprint arXiv:1804.02767.
- Bochkovskiy, A., Wang, C. Y., & Liao, H. Y. M. (2020). Yolov4: Optimal speed and accuracy of object detection. arXiv preprint arXiv:2004.10934.
- Jocher, G., et al. (2021). ultralytics/yolov5: v6.0 - YOLOv5n 'Nano' models, Roboflow integration, TensorFlow export, OpenCV DNN support (v6.0). Zenodo. <https://doi.org/10.5281/zenodo.5563715>

Reply:
Anyway, [ultralytics/yolov5](https://github.com/ultralytics/yolov5) is licensed under GPL-3.0. Is that OK?

Reply:
Addressed at https://github.com/tier4/AutowareArchitectureProposal.iv/pull/619/commits/3611fa7d486c8f7d83ca0cc003406b15aa42f07a